app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -54,6 +54,10 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
     }
   }
 
+  private fun doneClicked(): Next<PatientSummaryModel, PatientSummaryEffect> {
+    return dispatch(GoToHomeScreen)
```

Comment:
Same comment as earlier about the commit message not matching the actual change in this commit.